### ISSUE TITLE
Histograms showing contrasts between the different site parameters

### ISSUE THREAD

**Issue by aammd** [aammd/precipitation.experiment.schedule#2]
Hello @dsrivast ,
I've created a quick R script to make summaries of all the original data we have for rainfall amounts.  I think the next step is to select representative years from contrasting sites? [see mu and k figure here](https://github.com/aammd/precipitation.experiment.schedule/blob/master/site_comparison/site_comparison.md)

**Comment by aammd** [comment]
here is a [link to the figure](https://github.com/aammd/precipitation.experiment.schedule/blob/c3014f46671c0b50789f30338ee1a2c50c8dea3b/site_comparison/sitehists.pdf)

**Comment by dsrivast** [comment]
Yes, that is exactly what I wanted. Could you let me know which set of
plots is which site?

## 

Diane Srivastava
Professor
Dept of Zoology & Biodiversity Research Centre,
University of British Columbia
6270 University Blvd
Vancouver BC Canada V6T 1Z4
604-822-1350 (no voicemail)

**Comment by aammd** [comment]
whoops! latest commit adds titles

**Comment by dsrivast** [comment]
## did you push? I'm not seeing titles...

Diane Srivastava
Professor
Dept of Zoology & Biodiversity Research Centre,
University of British Columbia
6270 University Blvd
Vancouver BC Canada V6T 1Z4
604-822-1350 (no voicemail)

**Comment by aammd** [comment]
latest commit adds [pngs for representative years](https://github.com/aammd/precipitation.experiment.schedule/tree/master/site_comparison). (2x checked the link; should work :persevere: )
Would you like them in a different resolution, or with any other tweaks?
